Corrected a few typos; change the wording of the path planning section a little. Added myself to personnel.

generic-placeholder-name opened the pull request.

generic-placeholder-name committed:
Corrected a few typos; change the wording of the path planning section a little. Added myself to personnel.

mariojerez commented on path_planning.html:
```diff
@@ -54,12 +54,12 @@ <h2>Path Planning Team</h2>
+      <p>We frame the deer searching problem as an energy-efficient coverage path planning problem. Given a farm that we wish to surveil, we evenly distribute waypoints across the farm so that if a drone travels through all of the waypoints, then the entire area has been searched.</p>
       <img src="images/teams/path_planning/aco_sys_arch.jpg" alt="" />
-      <p>For each drone, we use <a href="https://ieeexplore.ieee.org/abstract/document/4129846" target="_blank">Ant Colony Optimization (ACO)</a> to generate energy-efficient flight path from its charging station, through all waypoints that aren't too close to obstacles such as trees or buildings, and back to its charging station. We <a href="https://ieeexplore.ieee.org/abstract/document/7989732" ...
+      <p>For each drone, we use <a href="https://ieeexplore.ieee.org/abstract/document/4129846" target="_blank">Ant Colony Optimization (ACO)</a> to generate energy-efficient flight path from its charging station, through all waypoints that aren't too close to obstacles such as trees or buildings, and back to its charging station. We <a href="https://ieeexplore.ieee.org/abstract/document/7989732" ...
```
```suggestion
      <p>For each drone, we use <a href="https://ieeexplore.ieee.org/abstract/document/4129846" target="_blank">Ant Colony Optimization (ACO)</a> to generate energy-efficient flight path from its charging station, through all waypoints that aren't too close to obstacles such as trees or buildings, and back to its charging station. We <a href="https://ieeexplore.ieee.org/abstract/document/7989732" target="_blank">approximate the energy cost</a> of a flight path by considering the distance and rotations required for a drone to execute the path. We constrain the flight paths of drones to travel along edges in straight lines. So, in order to ensure obstacle avoidance, we simply remove any edges that cross or come close to obstacles.</p>
```

mariojerez committed:
Update path_planning.html

We are removing the edge, not forbidding it from being created, since it was created and then later removed

mariojerez committed:
little cleanup in path planning page
